configure.ac

```diff
@@ -131,9 +131,10 @@ AC_ARG_ENABLE(gui-tests,
     [use_gui_tests=$enableval],
     [use_gui_tests=$use_tests])
 
+# Enable RapidCheck property-based tests
```

Comment:
I don't think we need the extra comment here. That string is in the arg description, and none of the other `AC_ARG_WITH` instances have comments.

Reply:
Thanks for reviewing. I saw that L247 (AC_ARG_WITH for sanitizers) has a comment as well as some of the AC_ARG_ENABLES. That said, ARG_WITH would normally be about building with a package, not enabling features, so L247 may not be a good example to follow. I'll remove and force push.